### PR TITLE
Start page

### PR DESCRIPTION
**Pull request by Junk-debug**

### CODE REVIEW

File: src/pages/StartPage.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Button } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import Layout from '../Layout';
+
+const StartPage: React.FC = () => {
+  const navigate = useNavigate();
+  return (
+    <Layout style={{ gap: '25px', transform: 'translateY(-5%)' }}>
```
Comment (Softjey) [inline src/pages/StartPage.tsx:8]: Fix styles, and your html layout to center it normally

File: src/pages/StartPage.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <h1>Examix</h1>
+      <div css={{ display: 'flex', flexDirection: 'column', gap: '15px' }}>
+        <Button
+          onClick={() => navigate('/login')}
```
Comment (Softjey) [inline src/pages/StartPage.tsx:12]: Check if MUI button is react memorized, and use useCallback if it is memorized, and do that everywhere

File: src/pages/StartPage.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+          css={{ width: '200px', height: '50px', fontSize: '1rem' }}
```
Comment (Softjey) [inline src/pages/StartPage.tsx:13]: Check MUI documentation and find better way to add your styles to button, fix it's everywhere

File: src/pages/StartPage.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+          variant="contained"
+        >
+          Login as teacher
+        </Button>
+        <Button
+          onClick={() => navigate('/join')}
+          css={{ width: '200px', height: '50px', fontSize: '1rem' }}
+          variant="contained"
+        >
+          Login as student
```
Comment (Softjey) [inline src/pages/StartPage.tsx:23]: Join the test

File: src/pages/StartPage.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      <div
+        css={{
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          gap: '25px',
+          height: '300px',
+        }}
+      >
+        <h1>Examix</h1>
```
Comment (Softjey) [inline src/pages/StartPage.tsx:18]: use your login header here, but with props, which disable backButton

File: src/pages/StartPage.tsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        <div css={{ display: 'flex', flexDirection: 'column', gap: '15px' }}>
+          <MainButton onClick={() => navigate('/login')} variant="contained">
```
Comment (Softjey) [inline src/pages/StartPage.tsx:20]: Change in all your components, navigate button to Link, it's more correct method
https://mui.com/material-ui/integrations/routing/

Comment (Softjey) [inline src/pages/StartPage.tsx:20]: Add const enum file with all routing paths and use them instead of string

File: src/pages/StartPage.tsx (hunk 5 above)
Comment (Junk-debug) [reply src/pages/StartPage.tsx:18]: i didnt get what you mean

Comment (Softjey) [reply src/pages/StartPage.tsx:18]: Use LoginHeader compenent instead of <h1>Examix</h1>, but make possibly to pass props, which disable BackButton

**Review by Softjey — APPROVED** [review]

**Pull request merged**